pkg/kms/localkms/localkms_writer.go

```diff
@@ -0,0 +1,69 @@
+/*
+ Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package localkms
+
+import (
+	"encoding/base64"
+	"fmt"
+	"io"
+
+	"github.com/google/tink/go/subtle/random"
+
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+)
+
+// newWriter creates a new instance of local storage key storeWriter in the given store and for masterKeyURI
+func newWriter(kmsStore storage.Store, masterKeyURI string) io.Writer {
+	return &storeWriter{
+		storage:      kmsStore,
+		masterKeyURI: masterKeyURI,
+	}
+}
+
+// storeWriter struct to store a keyset in a local store
+type storeWriter struct {
+	storage      storage.Store
+	masterKeyURI string
+	// KeysetID is set when Write() is called
+	KeysetID string
```

Comment:
accidental export?

Reply:
it's not accidental, this is how the writer exposes the KeySet ID after `Write()` is called, the KMS will access it for reference later (for getting or rotating a key)